### ISSUE TITLE
Help with testing on MSI X99A SLI PLUS

### ISSUE THREAD

**Issue by ulfbert-inc** [Dok-T/MS-7883#1]
Hey @Dok-T 
Thank you and all contributors so much for your work on these mods.

As an owner of MSI X99A SLI PLUS system I would like offer help testing mod. Let me know if you'd like that :)

**Comment by Dok-T** [comment]
Hello, yes why not!
Isn't you from winraid forum? Sorry for not replying that fast i'm currently working with this motherboard (the godlike) to build a NAS server for myself.

Regards,

DokT

**Comment by ulfbert-inc** [comment]
Sweet, I'll sign up for a forum account and will contact you there.

**Comment by Dok-T** [comment]
AFAIK this bios boot up on this boards but have issues like LAN because of xrong Intel GbE FWs

**Comment by ulfbert-inc** [comment]
Yeah I saw that and its a showstopper for  me as I rely on this boards built-in network. Perhaps something you (or myself with your guidance) could look into?

**Comment by Dok-T** [comment]
try to use get the UUID with UEFITool of network orom firmware and replace them with yours
